devtools/x/src/main.rs

```diff
@@ -3,13 +3,17 @@
 
 #![forbid(unsafe_code)]
 
+#[macro_use]
+extern crate clap;
+
```

Comment:
Don't use `extern crate` that's an old, deprecated syntax. You should be able to explicitly bring into scope the macros you want with `use clap::<macro>`